feed/history/boost_1_70_0.qbk

```diff
@@ -140,6 +153,16 @@
     is not documented in the reference section (i.e., it has semi-official status). 
   * Maintenance work.
   
+* [phrase library..[@/libs/multiprecision/ Multiprecision]:]
+  * Fix various conversion issues in the traits classes, check for compatibility with Boost.Optional.
+  * Prevent instantiation of `std::numeric_limits` on any old type when checking for convertibility.  See [@https://github.com/boostorg/multiprecision/issues/98 #98].
+  * Update variable precision code to accound for arbitrary precision integers.  See [@https://github.com/boostorg/multiprecision/issues/103 #103].
```

Comment:
s/accound/account/ ?

Reply:
Yup, created https://github.com/boostorg/website/pull/414 to fix the typo.
Thanks for the catch!